udc/src/test/java/io/crate/udc/ping/PingTaskTest.java

```diff
@@ -77,26 +78,49 @@ public void prepare() throws Exception {
 
         extendedNodeInfo = mock(ExtendedNodeInfo.class);
         when(extendedNodeInfo.networkInfo()).thenReturn(new ExtendedNetworkInfo(ExtendedNetworkInfo.NA_INTERFACE));
-        when(extendedNodeInfo.osInfo()).thenReturn(new ExtendedOsInfo(Collections.<String, Object>emptyMap()));
+        when(extendedNodeInfo.osInfo()).thenReturn(new ExtendedOsInfo(Collections.emptyMap()));
     }
 
     @Test
     public void testGetHardwareAddressMacAddrNull() throws Exception {
         PingTask pingTask = new PingTask(
-            mock(ClusterService.class),
-            mock(ClusterIdService.class),
+            clusterService,
+            clusterIdService,
             extendedNodeInfo,
-            "http://dummy");
+            "http://dummy",
+            Settings.EMPTY
+        );
         assertThat(pingTask.getHardwareAddress(), Matchers.nullValue());
     }
 
     @Test
+    public void testIsEnterpriseField() throws Exception {
+        PingTask pingTask = new PingTask(
+            clusterService,
+            clusterIdService,
+            extendedNodeInfo,
+            "http://dummy",
+            Settings.EMPTY
+        );
+        assertThat(pingTask.isEnterprise(), is("false"));
```

Comment:
probably, this code won't be even reached, coz licence.enterprise is a boolean setting